src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -85,7 +97,12 @@ export const GettingStarted = () => {
   }, [cloud, history]);
 
   const onSkip = async () => {
-    await guidedOnboardingService?.skipGuidedOnboarding();
+    try {
+      await guidedOnboardingService?.skipGuidedOnboarding();
+    } catch (error) {
+      // if the state update fails, it's safe to ignore the error
```

Comment:
When skipping the guided onboarding on the landing page, I think it's okay if the API request fails, so we can let the user leave the page.